python-flask-admin-portal-example/app.py

```diff
@@ -13,9 +12,12 @@
 lucide = Lucide(app)
 
 # WorkOS Setup
-workos.api_key = os.getenv("WORKOS_API_KEY")
-workos.project_id = os.getenv("WORKOS_CLIENT_ID")
-workos.base_api_url = "http://localhost:7000/" if DEBUG else workos.base_api_url
+base_api_url = "http://localhost:7000/" if DEBUG else None
```

Comment:
This feels kind of odd to me for a public example. I wonder if we should just use the `WORKOS_BASE_URL` environment variable here.

Reply:
Agreed. This is a simple change I can make to all of the sample apps.